### DISCUSSION TITLE
After buy manually from our own smart contract integration the nft is not showing in the Foundation profile

### DISCUSSION THREAD

**Discussion by empeje** [f8n/fnd-protocol#127] [Q&A] [answered]
Hi Team,

I'm currently integrating Foundation with our platform. We use direct integration using information in this repository to call the respected nft market smart contract. However when we buy item from our store, the nft is moving to new owner but int he foundation profile the nft is gone.

Here's the example:

* This is the NFT https://etherscan.io/nft/0x3b3ee1931dc30c1957379fac9aba94d1c48a5405/120213
* This is the transaction https://etherscan.io/tx/0xd8d0b158f58827ef56e84f03d23373f2be6aabd7c9450e6e2a34cc187c76fea3
* This is the foundation page https://foundation.app/0x005052a468069B40a239b9F0be7f6B98CdaCB096

Please advice,

Thank you

**Comment by HardlyDifficult** [comment, marked as answer]
Hey,
It looks like the NFT was sold on OpenSea shortly after the transaction on Foundation. It's now appearing on the latest owner's profile https://foundation.app/@Saucebook.Eth?tab=owned

You can see the transaction history on the etherscan page you had linked under item activity: https://etherscan.io/nft/0x3b3ee1931dc30c1957379fac9aba94d1c48a5405/120213

It seems like everything is working as expected here -- let us know if I overlooked something here.
